x-pack/plugins/observability_solution/apm/public/components/shared/try_it_button/index.tsx

```diff
@@ -156,25 +204,31 @@ export function TryItButton({
     );
   }
 
-  if (isFeatureEnabled) {
+  function HideThisButton() {
     return (
-      <EuiFlexGroup gutterSize="s" alignItems="center">
-        <Icon />
-        <Link />
-        <Popover />
-        <Loading />
-      </EuiFlexGroup>
+      <EuiFlexItem grow={false}>
+        <EuiLink
+          onClick={() => {
+            setShowFastFilterTryCallout(false);
+          }}
+        >
+          {i18n.translate('xpack.apm.tryItButton.dismiss', {
+            defaultMessage: 'Hide this',
+          })}
+        </EuiLink>
+      </EuiFlexItem>
     );
   }
 
   return (
     <EuiFlexGroup gutterSize="s" alignItems="center">
-      <TryItBadge />
+      {isFeatureEnabled ? null : <TryItBadge />}
       <Icon />
-      <PromoLabel />
-      <Popover />
+      {isFeatureEnabled ? null : <PromoLabel />}
       <Link />
+      <Popover />
       <Loading />
+      <HideThisButton />
```

Comment:
Hey @cauemarcondes - yeah, was thinking the same slightly.

In short, we want people to be able to get rid of this message if they want (i.e. dismiss it).

Is there a simple way we can allow people to 'kill this' so it won't keep showing?

Reply:
We can move the link to the tooltip:
WDYT?
<img width="582" alt="Screenshot 2024-04-24 at 19 39 13" src="https://github.com/elastic/kibana/assets/55978943/2326406e-4f59-4304-b628-30178d426314">

cc: @kkurstak

Reply:
Perhaps something like dismissible callouts?  https://eui.elastic.co/#/display/callout#dismissible-callouts. (it probably wasn't built for this purpose though)

Reply:
@roshan-elastic and I decided this:
<img width="450" alt="Screenshot 2024-04-25 at 13 46 08" src="https://github.com/elastic/kibana/assets/55978943/34c650ae-d2b3-4178-9fd5-54da2b40ace6">

@kpatticha 👇🏻
>On disable and hide, do you clear our the search input?
On disable yes, I'm resetting the search field when the user disables the feature.
On hide, there's no need to reset..

Reply:
Just out of curiosity: Why do we need both `Try it` and the tech preview badges? Also, if we need the tech preview badge, why don't we use it to show the tooltip, instead of having the `i` icon?

<img width="291" alt="image" src="https://github.com/elastic/kibana/assets/2767137/d64b8159-9801-4131-88fe-bd8d35774186">

Reply:
Good Q @crespocarlos...

Here's my opinion:

- Tech preview badge looks like it has a standard messaging associated with it around the UI so needs its own tooltip?
- The separate (i) tooltip we've added gives more information about the actual feature
- 'Try it'  is designs to encourage users to try it...the tech preview badge is to denote it's use at your own risk...I think they're two different things.

Reply:
Thanks @roshan-elastic , I'm not suggesting to change anything, but I'm just curious. 

> Tech preview badge looks like it has a standard messaging associated with it around the UI so needs its own tooltip?

Just thinking out loud here, could you use the standard tech preview messaging (which we don't seem to be using in this case) to create a popover that contains both that and the `give feedback` link and is activated when hovering over the tech preview badge?

> The separate (i) tooltip we've added gives more information about the actual feature

IMHO The CTA is self-explanatory. Also, the fast search input tells the user what they can search for.

Reply:
> Just thinking out loud here, could you use the standard tech preview messaging (which we don't seem to be using in this case) to create a popover that contains both that and the give feedback link and is activated when hovering over the tech preview badge?

Good spot @crespocarlos - I didn't realise the standard message wasn't there.

> IMHO The CTA is self-explanatory. Also, the fast search input tells the user what they can search for.

That's a fair point. I think these should separate though as I think tech preview icon would be where someone would hover over to learn more about 'tech preview' - I would expect the (i) to be where I learn more about the feature

@kkurstak - not sure on best practice here...do you think we should keep a separate (i) for the tooltip to explain it or should it be rolled into the tech preview badge tooltip/popover?

Reply:
I think it makes sense, as the tooltip will stay there, even when the tech-preview icon will at one point dissapear.